Apps/WebClient/src/ClientApp/test/home.test.ts

```diff
@@ -1,10 +1,16 @@
 import { shallowMount, createLocalVue } from '@vue/test-utils'
 import HomeComponent from '@/views/home.vue'
 
-describe('HomeComponent', () => {
+describe('Home view', () => {
+  const localVue = createLocalVue();
+  const wrapper = shallowMount(HomeComponent, { localVue });
+
   test('is a Vue instance', () => {
-    const localVue = createLocalVue()
-    const wrapper = shallowMount(HomeComponent, { localVue })
-    expect(wrapper.isVueInstance()).toBeTruthy()
+    expect(wrapper.isVueInstance()).toBeTruthy();
+  })
+
+  const expectedH1Text = "Health Gateway";
```

Comment:
nit: having the expect value inside the test itself unless is reused somewhere else